src/pages/Homepage/MonthlyGoodHomepage.vue

```diff
@@ -5,12 +5,21 @@
 				<div
 					class="small-12 medium-10 large-6 xlarge-5 small-order-2 large-order-1 columns"
 				>
-					<img
-						v-if="heroImage.url"
-						class="featured-loans__img"
-						:src="heroImage.url"
-						:alt="heroImage.description"
+					<router-link
+						:to="`/lend-by-category`"
```

Comment:
loan card should link to 'lend-by-category' - final tracking params on this page tbd